manifests/slave.pp

```diff
@@ -159,7 +159,7 @@
   $quoted_ui_pass = shellquote($ui_pass)
 
   if $labels {
-    if is_array($labels) {
+    if $labels =~ Array {
       $_combined_labels = hiera_array('jenkins::slave::labels', $labels)
```

Comment:
I do wonder what this `hiera_array` adds here, but that can be done in a separate PR.